Assets/MixedRealityToolkit.Providers/OpenVR/MixedReality.Toolkit.Providers.OpenVR.nuspec

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>Microsoft.MixedReality.Toolkit.Services</id>
+    <id>Microsoft.MixedReality.Toolkit.Unity.Providers.OpenVR</id>
     <version>$version$</version>
     <authors>Microsoft</authors>
-    <owners>Microsoft,MixedReality</owners>
-    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/License.txt</licenseUrl>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/License.txt</licenseUrl>
     <projectUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity</projectUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>The Mixed Reality Toolkit is a collection of scripts and components intended to accelerate development of applications targeting Microsoft HoloLens and Windows Mixed Reality headsets.</description>
+    <description>The Mixed Reality Toolkit OpenVR provider enables mixed reality applications to target the OpenVR platform and devices.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <releaseNotes>$releaseNotes$</releaseNotes>
-    <tags>Unity MixedReality</tags>
```

Comment:
Looks like some dependencies might be missing for the providers. Unity errors from installing this package, with similar ones for the other packages:

```
Assembly has reference to non-existent assembly 'Microsoft.MixedReality.Toolkit' (Assets/Packages/Microsoft.MixedReality.Toolkit.Unity.Providers.OpenVR.0.0.0/MixedRealityToolkit.Providers.OpenVR.asmdef)
```